src/backend/joanie/payment/models.py

```diff
@@ -344,12 +344,25 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+class CreditCardManager(models.Manager):
+    """Custom manager for `CreditCard` model"""
+
+    def get_card_for_owner(self, pk, user_id):
+        """
+        Retrieve a main credit card for a given owner.
+        This method attempts to retrieve a credit card that matches the given `card_id` and
+        `user_id`, and is marked as the main card. If no such card exists, it returns None.
```

Comment:
Your docstring is wrong. If no card exists, a `CreditCardDoesNotExist` exception will be raised.